task.go

```diff
@@ -116,6 +116,8 @@ func (r *marathonClient) KillApplicationTasks(id string, opts *KillApplicationTa
 //	opts:		KillTaskOpts request payload
 func (r *marathonClient) KillTask(taskID string, opts *KillTaskOpts) (*Task, error) {
 	appName := taskID[0:strings.LastIndex(taskID, ".")]
+	appName = strings.Replace(appName, "_", "/", -1)
```

Comment:
I'm not entirely sure whether we should duplicate the ID validation logic in the library, my reasoning here being that it will increase coupling and require us to watch out for changes in this regard. It wouldn't be the first unannounced breaking change that the Marathon project does.

That's just my opinion though. Feel free to open a new issue on the subject to discuss.